src/mechanics/constitutive/laws/LinearElasticInhomogeneous.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+
+#include "mechanics/constitutive/laws/LinearElasticEngineeringStress.h"
+#include "mechanics/constitutive/ConstitutiveBase.h"
+#include "mechanics/constitutive/staticData/IPConstitutiveLawWithoutData.h"
+
+namespace NuTo
+{
+
+class LinearElasticInhomogeneous: public LinearElasticEngineeringStress
```

Comment:
Nothing. I think it is still useful. Will move it to a seperate commit on master.